src/control/cmd/dmg/pool_test.go

```diff
@@ -621,30 +621,33 @@ func TestPoolCommands(t *testing.T) {
 					ID:        "031bcaf8-f0f5-42ef-b3c5-ee048676dceb",
 					Rank:      0,
 					TargetIdx: []uint32{1},
+					Force:     false,
 				}),
 			}, " "),
 			nil,
 		},
 		{
 			"Exclude a target with multiple idx",
-			"pool exclude 031bcaf8-f0f5-42ef-b3c5-ee048676dceb --rank 0 --target-idx 1,2,3",
+			"pool exclude -f 031bcaf8-f0f5-42ef-b3c5-ee048676dceb --rank 0 --target-idx 1,2,3",
```

Comment:
This maintains previous behavior, but we should have some test to verify force=False behaves as expected